tests/conftest.py

```diff
@@ -40,6 +41,17 @@ def get_data_frame(path, *paths, **kwargs):
         fullpath = os.path.join(DATA_DIR, path, *paths)
         return pd.read_csv(fullpath, **kwargs)
 
+    @staticmethod
+    def get_schema_explorer(path, *paths):
+        fullpath = Helpers.get_data_path(path, *paths)
+        
+        if fullpath is None:
+            return SchemaExplorer()
```

Comment:
I'm worried that `get_data_path()` doesn't ever return `None`. You might want to use the following approach: 

```suggestion
        if path is None:
            return SchemaExplorer()
            
        fullpath = Helpers.get_data_path(path, *paths)
```

Reply:
`path=None` as default?